src/System.Linq.Expressions/tests/Dynamic/DynamicObjectTests.cs

```diff
@@ -0,0 +1,791 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Linq.Expressions;
+using Microsoft.CSharp.RuntimeBinder;
+using Xunit;
+
+namespace System.Dynamic.Tests
+{
+    public class DynamicObjectTests
+    {
+#pragma warning disable 649
```

Comment:
Ironically, I didn't include the actual use of it I had intended, and so fell for the exact problem 649 is to advise about. It'll still be raised when I fix that though as the compiler won't realise a use through `dynamic` is a use, so I'll add a comment to the suppression.